### DISCUSSION TITLE
Bun v1.1.22

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#13155] [Announcements]
To install Bun v1.1.22:
```bash
curl -fsSL https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

Windows:
```bash
powershell -c "irm bun.sh/install.ps1|iex"
```

To upgrade to Bun v1.1.22:
```bash
bun upgrade
```

### Thanks to 11 contributors!

- [@billywhizz](htts://github.com/billywhizz)
- [@cirospaciari](htts://github.com/cirospaciari)
- [@dariushalipour](htts://github.com/dariushalipour)
- [@dylan-conway](htts://github.com/dylan-conway)
- [@Electroid](htts://github.com/Electroid)
- [@guest271314](htts://github.com/guest271314)
- [@huseeiin](htts://github.com/huseeiin)
- [@Jarred-Sumner](htts://github.com/Jarred-Sumner)
- [@m1212e](htts://github.com/m1212e)
- [@nektro](htts://github.com/nektro)
- [@paperdave](htts://github.com/paperdave)
- [@pythonmcpi](htts://github.com/pythonmcpi)

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.1.22'>Bun v1.1.22</a>.</em>

**Comment by stamminator** [comment]
I'm stoked about the `NODE_EXTRA_CA_CERTS` support! I previously wasn't able to use Bun at work due to our network using intermediate certificates. Time to see if Bun works for our web app build/bundle pipelines and can hopefully replace Node.

**Comment by Nanome203** [comment]
powershell -c "irm bun.sh/install.ps1|iex" installs v1.1.22 canary build, not stable one

**Reply by Jarred-Sumner** [reply]
We switched to a new CI and made a mistake
